hasura/metadata/databases/default/tables/public_circles.yaml

```diff
@@ -74,6 +81,30 @@ array_relationships:
         name: users
         schema: public
 select_permissions:
+- permission:
+    columns:
+    - alloc_text
+    - default_opt_in
```

Comment:
Are we missing the `discord_webhook` col for select permissions?  There is a potential security issue if `discord_webhook` is misused though

Reply:
Yea I don't think third parties should be able to read the discord webhook, I only allowed writing it so discord bot can automatically configure the webhook